kmm/kmm-post-test-fixtures/src/commonMain/kotlin/com/gchristov/newsfeed/kmmposttestfixtures/FakePostRepository.kt

```diff
@@ -19,22 +20,22 @@ class FakePostRepository(
         }
     }
 
-    override suspend fun post(postId: String, postMetadataFields: String): DecoratedPost {
-        return postResponse.execute(requireNotNull(post))
+    override suspend fun post(postId: String, postMetadataFields: String): Post {
+        return postResponse.execute(requireNotNull(post?.raw))
```

Comment:
`FakePostRepository` can now take in a `Post` as a parameter rather than `DecoratedPost` and we could update all this in a future PR where we address the caching to be able to store decorations 👍 

Not a huge deal but mostly for consistency. Up to you

Reply:
I think future PR to start closing this one... still have to do this one for iOS. But definitely, happy to keep adding improvements to learn much more

Reply:
Actually may not... as you mentioned in the other comment. Will come back

Reply:
Related to other comments, I think it's better leaving this for another PR. This is a monster one and these are even more changes. I'll try and keep next ones smaller :)